### PR TITLE
NPM deploy

### LINKED ISSUE #206: Add publish instructions to README.md
and give me access to publish, please :)

I suppose it is:
* Update the version in `package.json` to match the latest version in CHANGES.md, then run
```
npm run prepublish
npm publish
```

### PR DESCRIPTION
**Pull request by shehzan10**
Fixes #206
Includes #208

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,38 @@ The documentation will be placed in the `doc` folder.
...
+## Deploying to npm
+
+* Proofread [CHANGES.md](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/CHANGES.md).
+* Update the `version` in [package.json](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/package.json) to match the latest version in [CHANGES.md](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/CHANGES.md).
+* Make sure to run the tests and ensure they pass.
+* If any changes are required, commit and push them to the repository.
+* Create and test the package.
+```
+## NPM Pack
+## Creates tar ball. Verify using 7-zip (or your favorite archiver).
```
Comment (lilleyse) [inline README.md]: Change "tar ball" to "tarball"

File: README.md [hunk 2]
```diff
@@ -91,6 +91,38 @@ The documentation will be placed in the `doc` folder.
...
+## If you find unexpected/unwanted files, add them to .npmignore, and then run npm pack again.
+npm pack
+
+## Test the package
+## Copy and install the package in a temporary directory
+mkdir temp && cp <tarball> temp/
+npm install --production <tarball>
+node -e "var test = require('glTF-pipeline');" # No output on success
```
Comment (lilleyse) [inline README.md]: Use lowercase `gltf-pipeline`.

**Comment by shehzan10** [comment]
@lilleyse resolved the suggestions.

**Comment by lilleyse** [comment]
Looks good!

**Pull request merged**